### ISSUE TITLE
Add support for custom escape characters

### ISSUE THREAD

**Issue by cdhunt** [replicatedhq/dockerfilelint#82]
This should not throw warnings around the LABEL (line4 and 5)

```dockerfile
# escape=`
FROM microsoft/nanoserver
SHELL ["powershell.exe", "-C"]
LABEL maintainer="me" `
 description="docker container"
```

Use of the backtick is common in Windows dockerfiles because it aligns with the PowerShell line continuation character.

https://docs.docker.com/engine/reference/builder/#escape
